typescript/test/unit/proto.ts

```diff
@@ -15,8 +15,27 @@
 import * as assert from 'assert';
 
 import * as plugin from '../../../pbjs-genfiles/plugin';
-import { Proto } from '../../src/schema/proto';
+import { getHeaderParms } from '../../src/schema/proto';
 
 describe('schema/proto.ts', () => {
-  // TODO: test service augmentation
+  it('get header parameters from http rule', () => {
```

Comment:
nit: it's a tradition to start test descrptions with "should":

```js
it('should get header parameters from http rule', () => { ... });
```

that way it reads "it should ..." like as if it is written in English.

Reply:
...or just `it('gets header parameters from http rule')`, `it('works with no parameters')`, etc. - just make it a real sentence.

Reply:
Sounds reasonable, made changes